community/joystick.py

```diff
@@ -0,0 +1,113 @@
+import pyglet.input
+import math
+import hit
+import time
+import random
+
+class Joystick_controller:
```

Comment:
Perhaps in the future this could inherit from some `Controller` class?

Reply:
Yeah